test.py

```diff
@@ -0,0 +1,30 @@
+from GeneratorPgn import IteratorPgn, Preprocessor, ArrayToFen

```

Comment:
if you'd use a Python module for unit tests (https://docs.python.org/3/library/unittest.html), you would be able to write automatical checks like "now let's assert that correctly restored prercentage is equal to 100", not print values to be looked at